app/src/main/java/com/ustwo/boilerplate/feature/helloworld/HelloWorldActivity.java

```diff
@@ -10,6 +14,12 @@
     implements HelloWorldPresenter.View {
 
   @Inject HelloWorldPresenter presenter;
+  @BindView(R.id.textView_helloWorld) TextView helloWorldView;
+
+  @Override
+  protected void onInitialize() {
+    ButterKnife.bind(this);
```

Comment:
So it could go in `BaseActivity.onCreate` but might not be appropriate for all cases. In older builds of Butterknife calling Bind in a class that had nothing to bind would cause a crash. It can't go in `HelloWorldActivity.onCreate` since it would occur after the presenter is notified.

Let's try and think of a simpler but useful example for this :)